specs/unified_kernel_image.md

```diff
@@ -53,14 +53,15 @@ UKIs consist of the following resources:
 * Optionally, a microcode initrd in the `.ucode` section, to be handed to the kernel before any other initrd.
 * Optionally, a splash image to bring to screen before transitioning into the Linux kernel, in the `.splash` section.
 * Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtb` section. If multiple `.dtb` sections exist then one of them is selected according to an implementation-specific algorithm.
```

Comment:
please change the spec on `.dtb`to say that `.dtb` may only appear at most once. i.e. "one or more" → "a".

And drop the comments about sepearate dtb sections, and about the implementation specific algorithms

Reply:
That is, revert 5fab44aca328875f8a490c9e1b19752b03189fbc?

Reply:
Okay, it doesn't revert cleanly, but the current revision of this change basically includes a partial revert and changes ".dtb" to ".dtbauto" in a few places.